### PR TITLE
addressed points in issue 80, additional description for rotated pole…

### PR DESCRIPTION
**Pull request by balazagi**
…, guessing bounds and area weighting concepts

### CODE REVIEW

File: notebooks/CSSP_20CRDS_Tutorials/tutorial_2_data_preparation.ipynb [hunk 1]
```diff
@@ -373,7 +373,21 @@
     "\n",
     "To make your analysis faster and easier, you can extract a smaller part of the model domain. In these examples we will work with Shanghai, but you can choose any region you want. \n",
     "\n",
-    "<b>Note:</b> The original model data is on a rotated pole grid system and we need to the extract_rot_method() in order to get the data on to a regular grid system"
+    "<b>Note:</b> The original model data is on a rotated pole grid system which as seen in this diagram its coordinates are relative to the standard pole so we need to first get our data on to a standard grid system. We use the *extract_rot_method()* to do this."
```
Comment (nhsavage) [inline notebooks/CSSP_20CRDS_Tutorials/tutorial_2_data_preparation.ipynb:376]: I suggest: 
The original model data is on a rotated pole grid system, as show in the diagrams below. The x and y coordinates are not true latitude and longitude so to extract a latitude-longitude box, we use the *extract_rot_cube()* function to do this. It works by first calculating the true latitude and longitue of each grid cell and uses these to select which are in the area of interest.

File: notebooks/CSSP_20CRDS_Tutorials/tutorial_2_data_preparation.ipynb [hunk 2]
```diff
@@ -536,9 +550,11 @@
    "metadata": {},
    "source": [
     "### 2.1 Calculating mean, max, min\n",
-    "In this section we will use **iris.analysis** method to calculate basic mean, min and max values\n",
+    "In this section we will use **iris.analysis** method to calculate basic mean, min and max values. But before we do this we need to undrestand two important concepts/techniques that is used in the analysis code to follow:\n",
```
Comment (nhsavage) [inline notebooks/CSSP_20CRDS_Tutorials/tutorial_2_data_preparation.ipynb:553]: undrestand -> understand 
that is -> that are

File: notebooks/CSSP_20CRDS_Tutorials/tutorial_2_data_preparation.ipynb [hunk 3]
```diff
@@ -536,9 +550,11 @@
...
+    "1. Before we can do any analysis on spatial coordinates we need to be able to calculate the area of each grid box. and for this we need to know the boundaries of each grid box. If the longitude and latitude bounds are not defined in the cube we can guess the bounds based on the coordinates point values and that is what the *guess.bounds()* function does in the code below.\n",
```
Comment (nhsavage) [inline notebooks/CSSP_20CRDS_Tutorials/tutorial_2_data_preparation.ipynb:554]: 1 When we calculate area averages,  we need to

**Review by nhsavage — CHANGES_REQUESTED** [review]
some suggested improvemetns

**Comment by balazagi** [comment]
updated with suggested changes please review again.

**Review by nhsavage — APPROVED** [review]

**Pull request merged**